packages/blocks/src/page-block/doc/doc-page-block.ts

```diff
@@ -511,6 +511,25 @@ export class DocPageBlockComponent extends BlockElement<
       ) {
         return;
       }
+
+      // Click on blank area
```

Comment:
I think you should extract the onclick handler into sub methods. Simply adding logic here makes it looks verbose.